public/form/css/styles.css

```diff
@@ -616,6 +616,17 @@ hr {
   width: 100%;
 }
 
+.attribution-picture{
+  text-align:right;
+}
+
+.attribution-picture span{
+    color: #fff;
+    font-family: "Roboto Slab", serif;
```

Comment:
this is not being picked up - my browser shows me the attribution in times new roman